LDAR_Sim/src/file_processing/output_processing/output_constants.py

```diff
@@ -0,0 +1,92 @@
+class ProgTsConstants:
+    TITLE = "{name_str} Emissions Timeseries"
+    X_AXIS_TITLE = "Date"
+    Y_AXIS_TITLE = "Daily Emissions (kg Methane)"
+    FILENAME = "Timeseries"
+
+
+TS_SUMMARY_FILENAME = "daily_summary_stats.csv"
+EMIS_SUMMARY_FILENAME = "emissions_summary_stats.csv"
+TRUE_VS_ESTIMATED_PERCENT_DIFF_PLOT = "True_vs_Estimated_Emissions_percent_differences.png"
+TRUE_VS_ESTIMATED_RELATIVE_DIFF_PLOT = "True_vs_Estimated_Emissions_relative_differences.png"
+TRUE_AND_ESTIMATED_PAIRED_EMISSIONS_DISTRIBUTION_PLOT = (
+    "True_and_Estimated_Paired_Emissions_Distribution.png"
+)
+SUMMARY_PROGRAM_PLOTS_DIRECTORY = "program_summary_plots"
+
+
+class ESTIMATE_VS_TRUE_PLOTTING_CONSTANTS:
+    TRUE_EMISSIONS_SUFFIX = '"True" Total Emissions (Kg Methane)'
+    ESTIMATED_EMISSIONS_SUFFIX = '"Estimated" Total Emissions (Kg Methane)'
+    HISTOGRAM_BIN_WIDTH = 8.0
```

Comment:
Should this be something that's hard set, or something that users could eventually edit-be able to set?

Reply:
I think this should be semi settable by users - In the sense we give them options - not let them set any number. Will add a TODO